docker-compose.yml

```diff
@@ -8,6 +8,15 @@ services:
     volumes:
       - /var/lib/postgresql
     command: echo true
+    
+  nginx:
+    build: .
+    ports:
+        - "80:80"
```

Comment:
The web service has already taken port 80 in host. The correct way would be to assign Nginx the port 80 in host (like above). Nginx would directly serve the static assets and redirect other requests to the web (gunicorn server) service. These redirects would be inter-container requests. 

Another way of doing it could be just using the nginx server installed inside the web service. Run it on 5000 port (port on which gunicorn now runs on), assign gunicorn to some other port, and do the same redirection like above. In this nginx and gunicorn would run on the same container.

Gunicorn shouldn't be used to serve static assets in production. Nginx can be used as a proxy server to serve static content directly and pass other requests to gunicorn. I think we should discuss it in the chat first. @amitkumarj441 @aviaryan